pkg/machines/components/vcpuModal.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { Button, Modal } from 'patternfly-react';
 import cockpit from 'cockpit';
 
+import { ModalError } from './notification/inlineNotification.jsx';
```

Comment:
This commit, or one of the previous ones, should add a `Fixes #10526`.

Reply:
Was not sure if I should add ```Fixes #``` because there are some extra comments from garret there that I have not addressed in this PR since it's harder to fix them and/or not supported fully yet in pf-react. But I can close it and have it just in mind for future.

Reply:
Added ```Fixes #```